ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -154,6 +164,29 @@ class NavigationControllerDemoController: DemoController {
 
         return controller
     }
+    
+    private func changeStyle(in navigationItem: UINavigationItem) {
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+            var newStyle = navigationItem.navigationBarStyle
```

Comment:
you can do let newStyle: NavigatonBarStyle and then in the switch statement it will assign the value.